### DISCUSSION TITLE
How to use local volume with milvus running with k3s (with helm)? So it uses a node's hard drive as a storage.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22890] [Q&A and General discussion]
How to use local volume with milvus running with k3s (with helm)? So it uses a node's hard drive as a storage.

**Comment by LoveEachDay** [comment]
You'd first check the provisioned storageclass like this:
```
kubectl get storageclass
```
whether it has a storageclass named `local-path`. If not, try to create one following the instructions at here: https://github.com/rancher/local-path-provisioner
